CHANGELOG.md

```diff
@@ -3,6 +3,18 @@ All notable changes to this project will be documented in this file.
 
 This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
+## [vector-0.20.0] - 2023-02-27
+
+### Vector
+
+#### Bug Fixes
+
+- Bump Vector to 0.27.1 (#284) ([bca819f](https://github.com/vectordotdev/helm-charts/commit/bca819f25e4ee3733b9f20d8ddfc7bf35ae5db1f))
```

Comment:
I'm only noticing this now, but shouldn't 0.19.2 have this note and this PR have the 0.28 note?